### PR TITLE
added-agile-questions

### PR DESCRIPTION
**Pull request by KmKalpana**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by KmKalpana** [comment]
I have added agile-quiz-questions

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+  {
+    Questions:
+      "What is an iterative development methodology, where both development and testing activities are concurrent.",
+    Answer: "Agile",
+    Distractor1: "Bugzilla",
+    Distractor2: "Apache",
+    Distractor3: "Espresso",
+    Explanation:
+      "Agile is an iterative development methodology, where both development and testing activities are concurrent.",
+    Link: "https://www.agilealliance.org/agile101/",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:254]: Please only use freeCodeCamp resources for the link field.

File: src/data/agile-quiz.ts [hunk 2]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+  },
+  {
+    Questions: "FFD Stands for?",
+    Answer: "Feature Driven Development",
+    Distractor1: "Feature Design Development",
+    Distractor2: "Feature Driven Design",
+    Distractor3: "Feature Driven Difference",
+    Explanation:
+      "Feature Driven Development (FDD) involves designing and building features. The difference between FDD and other Agile Development Methodologies is that the features are developed in specific and short phases separately.",
+    Link: "https://www.planview.com/resources/articles/fdd-agile/",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:264]: Please only use freeCodeCamp resources for the link field.

File: src/data/agile-quiz.ts [hunk 3]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
       "Story points are used to estimate the amount of effort required to deliver a user story for a project.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/#story-points-and-complexity-points",
   },
+  {
+    Questions:
+      "What is an iterative development methodology, where both development and testing activities are concurrent.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:247]: This wording is a little to close to here
https://sceweb.sce.uhcl.edu/helm/WEBPAGE-AgileTesting


```suggestion
      "What is the name for the process that focuses on continuously building, testing, and gathering feedback for software products?",
```

File: src/data/agile-quiz.ts [hunk 4]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+    Answer: "Agile",
+    Distractor1: "Bugzilla",
+    Distractor2: "Apache",
+    Distractor3: "Espresso",
+    Explanation:
+      "Agile is an iterative development methodology, where both development and testing activities are concurrent.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:253]:
```suggestion
      "Agile is a project management process that focuses on continuously building, testing, and gathering feedback for software products.",
```

File: src/data/agile-quiz.ts [hunk 5]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+    Link: "https://www.agilealliance.org/agile101/",
+  },
+  {
+    Questions: "FFD Stands for?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:257]:
```suggestion
    Questions: "In Agile, what does FFD stand for?",
```

File: src/data/agile-quiz.ts [hunk 6]
```diff
@@ -242,6 +242,27 @@ const agileQuiz = [
...
+    Answer: "Feature Driven Development",
+    Distractor1: "Feature Design Development",
+    Distractor2: "Feature Driven Design",
+    Distractor3: "Feature Driven Difference",
+    Explanation:
+      "Feature Driven Development (FDD) involves designing and building features. The difference between FDD and other Agile Development Methodologies is that the features are developed in specific and short phases separately.",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:263]:
```suggestion
      "Feature Driven Development (FDD) is the practice of incrementally developing and frequently releasing new features for software products.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**